bakery/src/tasks/look-up-book.js

```diff
@@ -30,15 +34,28 @@ const task = (taskArgs) => {
           exec 2> >(tee book/stderr >&2)
           tail ${inputSource}/*
           cp ${inputSource}/id book/job_id
-          cp ${inputSource}/collection_id book/collection_id
           cp ${inputSource}/version book/version
-          cp ${inputSource}/collection_style book/style
-          cp ${inputSource}/content_server book/server
-          wget -q -O jq 'https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64' && chmod +x jq
-          server_shortname="$(cat ${inputSource}/job.json | ./jq -r '.content_server.name')"
-          echo "$server_shortname" >book/server_shortname
-          pdf_filename="$(cat book/collection_id)-$(cat book/version)-$(cat book/server_shortname)-$(cat book/job_id).pdf"
-          echo "$pdf_filename" > book/pdf_filename
+          if [[ "$CONTENT_SOURCE" = archive ]]; then
+            cp ${inputSource}/collection_id book/collection_id
+            cp ${inputSource}/collection_style book/style
+            cp ${inputSource}/content_server book/server
+            wget -q -O jq 'https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64' && chmod +x jq
+            server_shortname="$(cat ${inputSource}/job.json | ./jq -r '.content_server.name')"
+            echo "$server_shortname" >book/server_shortname
+            pdf_filename="$(cat book/collection_id)-$(cat book/version)-$(cat book/server_shortname)-$(cat book/job_id).pdf"
+            echo "$pdf_filename" > book/pdf_filename
+          else
+            if [[ "$CONTENT_SOURCE" = git ]]; then
+              cp ${inputSource}/collection_style book/style
```

Comment:
Since this is shared in both cases, can it be pulled out similar to `job_id` and `version` above in L36-37?